packages/fx-core/src/plugins/solution/fx-solution/arm.ts

```diff
@@ -822,3 +841,22 @@ function formattedDeploymentName(failedDeployments: string[]): Result<void, FxEr
   );
   return err(returnUserError(returnError, "Solution", "ArmDeploymentFailed", ArmHelpLink));
 }
+
+function formattedDeploymentError(deploymentError: any): any {
+  if (deploymentError.subErrors) {
+    const result: any = {};
+    for (const key in deploymentError.subErrors) {
+      const subError = deploymentError.subErrors[key];
+      if (subError.inner) {
+        result[key] = formattedDeploymentError(subError.inner);
+      } else {
+        if (!subError.error?.message?.includes("Template output evaluation skipped")) {
```

Comment:
is there an error code for this? if yes, can we use the error code to filter the error? The major concern is that ARM may return different error message for a similar issue, though I'm not sure whether it will do this...

Reply:
The error code is a general code "DeploymentOperationFailed", which contains many detailed errors. It will filter else errors if we use this error code. I will add the error code as filter with error message as well.